web/README.desktop-app.md

```diff
@@ -94,4 +105,10 @@ Then run `yarn sign-dmg` from `web`. After this step finished, the dmg file is n
 
 ### Release the Windows App
 
-Submit the `AppX` file to the store. Upload the `.exe` file to GitHub release for downloading outside the store.
+Submit the `AppX` file at `web/build/appx/dolt.exe` to the store. To submit, choose DoltHub.Inc -> Apps and Games -> Dolt-Workbench to update.
```

Comment:
Submit the `AppX` file at `web/dist` to the store. To submit, choose DoltHub.Inc -> Apps and Games -> Dolt-Workbench to update.